platform/android/export/export.cpp

```diff
@@ -2625,19 +2662,29 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 				EditorNode::get_singleton()->show_warning(TTR("Building of Android project failed, check output for the error.\nAlternatively visit docs.godotengine.org for Android build documentation."));
 				return ERR_CANT_CREATE;
 			}
-			if (p_debug) {
-				src_apk = build_path.plus_file("build/outputs/apk/debug/android_debug.apk");
-			} else {
-				src_apk = build_path.plus_file("build/outputs/apk/release/android_release.apk");
+
+			List<String> copy_args;
+			String copy_command;
+			if (export_format == 1) {
+				copy_command = vformat("copyAndRename%sAab", build_type);
+			} else if (export_format == 0) {
+				copy_command = vformat("copyAndRename%sApk", build_type);
 			}
+			copy_args.push_back(copy_command);
+			print_line(p_path);
+			String Pexport_filename = p_path.get_file();
+			String Pexport_path = p_path.get_base_dir().replace("res://", "../../");
```

Comment:
Those two variables'names should follow the snake case style but that would make them look like they're method parameters (e.g: `p_export_filename`). Is the prefix `P` really needed?